test/e2e/logs_include_namespaces_test.go

```diff
@@ -0,0 +1,104 @@
+//go:build e2e
+
+package e2e
+
+import (
+	"net/http"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	kitk8s "github.com/kyma-project/telemetry-manager/test/testkit/k8s"
+	. "github.com/kyma-project/telemetry-manager/test/testkit/matchers/log"
+	"github.com/kyma-project/telemetry-manager/test/testkit/mocks/backend"
+	"github.com/kyma-project/telemetry-manager/test/testkit/mocks/loggen"
+	"github.com/kyma-project/telemetry-manager/test/testkit/periodic"
+	"github.com/kyma-project/telemetry-manager/test/testkit/verifiers"
+)
+
+var _ = Describe("Logs Include Namespaces", Label("logs"), Ordered, func() {
+	const (
+		mockNs          = "log-include-namespaces-mocks"
+		systemNamespace = "kyma-system"
```

Comment:
Please use `kitkyma.SystemNamespaceName` in both new test suites. And maybe a constant for `kube-system` in kitkyma?

Reply:
Done